Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_037.SqlCredentialsValidator.cs

```diff
@@ -38,13 +38,10 @@ public SqlCredentialsValidator()
 
         protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary<string, FlexMatch> groups)
         {
-            if (!groups.TryGetNonEmptyValue("id", out FlexMatch id) ||
-                !groups.TryGetNonEmptyValue("host", out FlexMatch host) ||
-                !groups.TryGetNonEmptyValue("secret", out FlexMatch secret) ||
-                !groups.TryGetNonEmptyValue("resource", out FlexMatch resource))
-            {
-                return ValidationResult.CreateNoMatch();
-            }
+            FlexMatch id = groups[nameof(id)];
```

Comment:
>FlexMatch [](http://example.com/codeflow?start=12&length=9)

here is the code clean-up that the new optionally provided group syntax permits us to do. @hulonjenkins